### PR TITLE
Fix for when the keyboard covers the Tool Bar

### LINKED ISSUE #535: FLEX Explorer could move its position if it's covered by keyboard
Currently the FLEX Explorer is showing behind the keyboard, which made it inaccessible. Ideally FLEX should listen to the keyboard frame changes and move itself up a bit to be visible.

### PR DESCRIPTION
**Pull request by ExTBH**
Resolves #535 
Tested only on the Simulator, with Xcode 13.4.1

### CODE REVIEW

File: Classes/ExplorerInterface/FLEXExplorerViewController.m [hunk 1]
```diff
@@ -375,6 +378,14 @@ - (CGRect)frameInLocalCoordinatesForView:(UIView *)view {
     return [self.view convertRect:frameInWindow fromView:nil];
 }
 
+- (void)keyboardWasShown:(NSNotification*)aNotification{
+    CGPoint kbFrame = [[[aNotification userInfo] objectForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue].origin;
+    
+    CGRect newToolbarFrame = self.explorerToolbar.frame;
+    newToolbarFrame.origin.y = kbFrame.y - newToolbarFrame.size.height;
```
[the remark this answers is not in the document]
Comment (ExTBH) [reply Classes/ExplorerInterface/FLEXExplorerViewController.m]: I've updated it, what does the change do? i haven't worked with frames and such idk

[the remark this answers is not in the document]
Comment (ExTBH) [reply Classes/ExplorerInterface/FLEXExplorerViewController.m]: i realized what you meant after i made the seconds commit, it feels like an idiot :) wish there's delete commits button
i made a check commit but saw you comment after, i'll add the animation sound good idea.
i'll go now i'll do it in ~30 minutes maybe

[the remark this answers is not in the document]
Comment (ExTBH) [reply Classes/ExplorerInterface/FLEXExplorerViewController.m]: I've added the animation, looks nice.

**Comment by NSExceptional** [comment]
Thanks!

**Pull request merged**